docs/operator-manual/argocd-cm.yaml

```diff
@@ -345,12 +345,6 @@ data:
   # This is to prevent the UI from becoming unresponsive when rendering a large number of logs. Default is 10.
   server.maxPodLogsToRender: "10"
 
-  # Application pod logs RBAC enforcement enables control over who can and who can't view application pod logs.
```

Comment:
Should we leave this section and just call out that the default is now true and maybe change the verbiage to say what happens when you disable it? 

Having entries defined here helps operators know what they can toggle and their default settings.